### PR TITLE
fix: duration doesn't get accurately compared in multi alb service base

### PR DESCRIPTION
**Pull request by markanderstam**
Duration wasn't getting compared correctly and prevented some values, eg between 5 and 9 or 500 and 999 (perhaps others as well), from getting used for `ApplicationMultipleTargetGroupsService.idleTimeout`.

Relates to https://github.com/aws/aws-cdk/pull/21584, that fixes this issue for ApplicationLoadBalancedService (however the problem description was not fully correct in regards to what range of values were getting denied).

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by markanderstam** [comment]
Exemption Request

**Review by scanlonp — CHANGES_REQUESTED** [review]
Looks good! We already have an idle timeout integ test ([here](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk-testing/framework-integ/test/aws-ecs-patterns/test/ec2/integ.multiple-application-load-balanced-ecs-service-idle-timeout.ts)) so changing the timeout values there to match your unit test would be prefered, but not strictly necessary.

File: packages/aws-cdk-lib/aws-ecs-patterns/lib/base/application-multiple-target-groups-service-base.ts [hunk 1]
```diff
@@ -585,7 +585,8 @@ export abstract class ApplicationMultipleTargetGroupsServiceBase extends Constru
   private validateLbProps(props: ApplicationLoadBalancerProps[]) {
     for (let prop of props) {
       if (prop.idleTimeout) {
-        if (prop.idleTimeout > Duration.seconds(4000) || prop.idleTimeout < Duration.seconds(1)) {
+        const idleTimeout = props.idleTimeout.toSeconds();
```
[the remark this answers is not in the document]
Comment (markanderstam) [reply packages/aws-cdk-lib/aws-ecs-patterns/lib/base/application-multiple-target-groups-service-base.ts]: I was not able to locate the error from the build failure. Thank you!

I will change the integration test as well. Did not realize there was one.

**Review by scanlonp — CHANGES_REQUESTED** [review]
Thanks for the integ test update!
Last thing, do not think we need to update yarn.lock. Mind removing the change from the PR unless it is necessary?

**Review by scanlonp — APPROVED** [review]
Wonderful. Thanks!

**Pull request merged**